en/lessons/basics/basics.md

```diff
@@ -9,269 +9,197 @@ Getting started, basic data types, and basic operations.
 
 ## Getting Started
 
-### Installing Elixir
+### Installing Haskell
 
-Installation instructions for each OS can be found on elixir-lang.org in the [Installing Elixir](http://elixir-lang.org/install.html) guide.
+The recommended installation instructions for each OS can be found at
+[ghcup](https://www.haskell.org/ghcup/).
 
-After Elixir is installed, you can easily find the installed version.
+You can check the version of the Haskell compiler installed by running
+`ghc --version`:
 
-    % elixir -v
-    Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
-
-    Elixir {{ site.elixir.version }}
+    % ghc --version
+    The Glorious Glasgow Haskell Compilation System, version {{ site.ghc.version }}
 
 ### Trying Interactive Mode
 
-Elixir comes with IEx, an interactive shell, which allows us to evaluate Elixir expressions as we go.
+Your installation should come with `ghci`, an interactive shell that allows you
+to try out code in real time.
 
-To get started, let's run `iex`:
+Follow this tutorial by running `ghci`:
 
-    Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+    GHCi, version {{ site.ghc.version }}: https://www.haskell.org/ghc/  :? for help
+    ghci>
 
-    Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
-    iex>
+To use `Text` in Haskell, type in:
 
-Note: On Windows PowerShell, you need to type `iex.bat`.
+```haskell
+ghci> :set -XOverloadedStrings
+ghci> import qualified Data.Text as T
+```
 
-Let's go ahead and give it a try now by typing in a few simple expressions:
+You can try out Haskell by typing in a few simple expressions:
 
-```elixir
-iex> 2+3
+```haskell
+ghci> 2 + 3
 5
-iex> 2+3 == 5
-true
-iex> String.length("The quick brown fox jumps over the lazy dog")
+ghci> 2 + 3 == 5
+True
+ghci> T.length "The quick brown fox jumps over the lazy dog"
 43
 ```
 
-Don't worry if you don't understand every expression yet, but we hope you get the idea.
+Perhaps you're able to figure out what some of these expressions mean.
 
 ## Basic Data Types
 
 ### Integers
 
-```elixir
-iex> 255
+Haskell has a bounded integer type called `Int`:
+
+```haskell
+ghci> 255 :: Int
 255
 ```
+Note: specify the type to use with `::`.
```

Comment:
This reads as if the reader must commit some action when the note merely provides information about type annotations.